deployment/GI05MOAS-GL363_Deploy.csv

```diff
@@ -1,5 +1,5 @@
-CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes,electrical.uid,assembly_template_revision
-AR30-03,,AR35-03,,GI05MOAS-GL363-00-ENG000000,3,1,2018-06-09T19:49:00,2019-06-19T07:18:24,CGVGI-05MOAS-00000,CGVEH-GLDROO-00363,CGCON-GLDROO-00363,59.9207,-39.3008,,1000,1000,Project glider with special sampling and modified optode location,,
-AR30-03,,AR35-03,,GI05MOAS-GL363-01-FLORDM000,3,1,2018-06-09T19:49:00,2019-06-19T07:18:24,CGVGI-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-FLORDM-03063,59.9207,-39.3008,,1000,1000,Project glider with special sampling and modified optode location,,
-AR30-03,,AR35-03,,GI05MOAS-GL363-02-DOSTAM000,3,1,2018-06-09T19:49:00,2019-06-19T07:18:24,CGVGI-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-DOSTAM-00341,59.9207,-39.3008,,1000,1000,Project glider with special sampling and modified optode location,,
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes,electrical.uid,assembly_template_revision
+AR30-03,,AR35-03,,GI05MOAS-GL363-00-ENG000000,3,1,2018-06-09T19:49:00,2019-06-19T07:18:24,CGVGI-05MOAS-00000,CGVEH-GLDROO-00363,CGCON-GLDROO-00363,59.9207,-39.3008,,1000,1000,Project glider with special sampling and modified optode location,,
+AR30-03,,AR35-03,,GI05MOAS-GL363-01-FLORDM000,3,1,2018-06-09T19:49:00,2019-06-19T07:18:24,CGVGI-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-FLORDM-03063,59.9207,-39.3008,,1000,1000,Project glider with special sampling and modified optode location,,
+AR30-03,,AR35-03,,GI05MOAS-GL363-02-DOSTAM000,3,1,2018-06-09T19:49:00,2019-06-19T07:18:24,CGVGI-05MOAS-00000,CGVEH-GLDROO-00363,CGINS-DOSTAM-00341,59.9207,-39.3008,,1000,1000,Project glider with special sampling and modified optode location; top mount,,

```

Comment:
Looks good.  But do all of the instrument need the note saying "...and modified optode location"?  Seems like that should be removed.